solana-programs/Dockerfile

```diff
@@ -27,8 +28,12 @@ RUN cd audius_eth_registry && \
     cd ../claimable-tokens/program && \
     cargo build-bpf && \
     cd ../cli && \
+    cargo build && \ 
+    cd ../../reward-manager/program && \
+    cargo build-bpf && \
+    cd ../cli && \
```

Comment:
I don't think we need to build cli twice?

Reply:
nvm, I misread this bit of code, resolved

Reply:
Maybe add a comment, or make this directory part easier to read?

Reply:
will do